release_notes/v1.1.md

```diff
@@ -0,0 +1,328 @@
+- Text-to-Speech: add [GPT-SoVITS microservice](https://github.com/opea-project/GenAIComps/pull/784)
+- Avatar Animation: add [Animation microservice](https://github.com/opea-project/GenAIComps/pull/775)
+- RAG: add [GraphRAG with llama-index microservice](https://github.com/opea-project/GenAIComps/pull/793)
+
+#### Enhanced GenAI Components
+- Asynchronous support for microservices ([28672956](https://github.com/opea-project/GenAIComps/commit/28672956), [9df4b3c0](https://github.com/opea-project/GenAIComps/commit/9df4b3c0), [f3746dc8](https://github.com/opea-project/GenAIComps/commit/f3746dc8))
+- Add vLLM backends for [summarization](https://github.com/opea-project/GenAIComps/commit/550325d8), [FAQ generation](https://github.com/opea-project/GenAIComps/commit/f5c60f10), [code generation](https://github.com/opea-project/GenAIComps/commit/24b9f03f), and [Agents](https://github.com/opea-project/GenAIComps/commit/4638c1d4)
+- Multimedia support ([29ef6426](https://github.com/opea-project/GenAIComps/commit/29ef6426), [baafa402](https://github.com/opea-project/GenAIComps/commit/baafa402))
+
+#### GenAIStudio
+[GenAI Studio](https://github.com/opea-project/GenAIStudio), a new project of OPEA, streamlines the creation of enterprise Generative AI applications by providing alternate to manual processes with a seamless, end-to-end solution. From GenAI app development and evaluation to performance benchmarking and deployment, GenAI Studio empowers developers to effortlessly build, test, optimize their LLM solutions and create the deployment package. Its intuitive no-code/low-code interface accelerates innovation, enabling rapid development and deployment of cutting-edge AI applications with unparalleled efficiency and precision.
+
+#### Enhanced Observability
+Observability offers real-time insights into component performance and system resource utilization. We enhanced this capability by monitoring [key system metrics](https://github.com/opea-project/GenAIInfra/pull/541), including [CPU](https://github.com/opea-project/GenAIInfra/pull/541), host memory, storage, network, and accelerators (such as [Intel Gaudi](https://github.com/opea-project/GenAIInfra/pull/541)), as well as tracking [OPEA application scaling](https://github.com/opea-project/GenAIInfra/pull/541).
+
+#### Helm Charts Support
+OPEA examples and microservices support Helm Charts as the packaging format on K8S. The newly supported examples include [AgentQnA](https://github.com/opea-project/GenAIInfra/commit/66de41c), [AudioQnA](https://github.com/opea-project/GenAIInfra/commit/9efacee),  [FaqGen](https://github.com/opea-project/GenAIInfra/commit/f847e05), [VisualQnA](https://github.com/opea-project/GenAIInfra/commit/b077d44) etc. The newly supported microservices include [chathistory, mongodb, prompt](https://github.com/opea-project/GenAIInfra/commit/881e2b5), and [Milvus for data-prep and retriever](https://github.com/opea-project/GenAIInfra/commit/d289b4e).
```

Comment:
s/K8S/Kubernetes (k8s)/

Instead of "etc.", couldn't this just list all the new examples?

IMHO links pointing to READMEs of each new helm chart, would be more user friendly than pointing to commits adding them.   Alternatively, those links could point to PRs adding the examples, so that one sees also rest of things related to PR.

This could also mention that Helm Charts have now option to get Prometheus metrics from the applications.

Reply:
Thanks for your suggestions. Updated.